acrobat/styles/styles.css

```diff
@@ -445,7 +464,7 @@ div.how-to ol li::before{
   left: 4px;
 }
 
-.accordion dt button {
+.accordion dt .accordion-trigger {
```

Comment:
Do we still need these accordion style overrides?

Reply:
@joaquinrivero do we? if